sql/glam-fenix-dev/glam_etl/firefox_desktop_glam_nightly__histogram_bucket_counts_v1/query.sql

```diff
@@ -43,7 +43,7 @@ build_ids AS (
     1,
     2
   HAVING
-    COUNT(DISTINCT client_id) > 10
+    COUNT(DISTINCT client_id) > 0
```

Comment:
This was caused by [2651](https://github.com/mozilla/bigquery-etl/pull/2651/files) which did not include this file.